datafusion/optimizer/examples/rewrite_expr.rs

```diff
@@ -0,0 +1,163 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I personally recommend putting this example into `datafusion/examples` rather than `datafusion/optimizer/examples` so that it is easier to find

Reply:
Done